src/EPR.Accreditation.Facade/Services/OverseasSiteService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using EPR.Accreditation.Facade.Common.Dtos;
+using EPR.Accreditation.Facade.Services.Interfaces;
+using EPR.Accreditation.Facade.Common.RESTservices.Interfaces;
+
+namespace EPR.Accreditation.Facade.Services
+{
+    public class OverseasSiteService : IOverseasSiteService
+    {
+        protected readonly IHttpOverseasSiteService _httpOverseasSiteService;
+
+        public OverseasSiteService(IHttpOverseasSiteService httpOverseasSiteService)
+        {
+            _httpOverseasSiteService = httpOverseasSiteService ?? throw new ArgumentNullException(nameof(httpOverseasSiteService));
+        }
+
+        public async Task<OverseasAddress> GetReprocessorDetails(
+            Guid accreditationExternalId,
+            Guid overseasSiteExternalId)
+        {
+            var overseasSite = await _httpOverseasSiteService.GetOverseasReprocessingSite(
+                accreditationExternalId,
+                overseasSiteExternalId);
+
+            if (overseasSite.OverseasAddress == null)
+                return null;
```

Comment:
StyleCop for Portal requires braces even for one line if stamenents. Let's continue that style

Reply:
Added